soc/riscv/riscv-privilege/telink_b9x/soc.c

```diff
@@ -4,11 +4,20 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 #include <sys.h>
+#elif CONFIG_SOC_RISCV_TELINK_B95
+#include <lib/include/sys.h>
```

Comment:
"lib/include" can be removed and we can use one incluse for all SoCs

Reply:
will add the lib/include  into cmake list .